pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1905,4 +1905,16 @@ self: super: {
   # https://github.com/enomsg/science-constants-dimensional/pull/1
   science-constants-dimensional = doJailbreak super.science-constants-dimensional;
 
+  # Test assets missing from sdist
+  # https://github.com/hadolint/language-docker/issues/63
+  language-docker = overrideSrc super.language-docker {
+    src = pkgs.fetchFromGitHub {
+      owner = "hadolint";
+      repo = "language-docker";
+      rev = "refs/tags/${super.language-docker.version}";
```

Comment:
```suggestion
      rev = super.language-docker.version;
```

Reply:
That actually does not work in this case.